### PR TITLE
Fixes #34315 - adds Oracle VIF hypervisor NIC filter

### PR DESCRIPTION
**Pull request by lzap**
SSIA

https://community.theforeman.org/t/puma-cluster-worker-using-huge-amounts-of-memory/26978

### CODE REVIEW

File: app/registries/foreman/settings/facts.rb [hunk 1]
```diff
@@ -36,6 +36,7 @@
       'vlinuxbr*',
       'vovsbr*',
       'br-int',
+      'vif*,',
```
Comment (ezr-ondrej) [inline app/registries/foreman/settings/facts.rb]: As already pointed out by `jjvcuyler` in the thread:
```suggestion
      'vif*',
```

**Review by ezr-ondrej — CHANGES_REQUESTED** [review]
I did not find enough info on what the vif* actually is, but seems to fix the issue for our users, so I'm in. But there is probably a typo. Could you please fix that ? :)

File: app/registries/foreman/settings/facts.rb (hunk 1 above)
Comment (lzap) [reply app/registries/foreman/settings/facts.rb]: Yeah typo, fixed.

**Review by ezr-ondrej — APPROVED** [review]

**Comment by ezr-ondrej** [comment]
Thanks @lzap ! :+1:

**Pull request merged**